packages/table/src/Cell/Cell.tsx

```diff
@@ -1,31 +1,35 @@
-import React from 'react';
+import React, { ForwardedRef } from 'react';
 
 import { LGRowData } from '../useLeafyGreenTable';
 
+import { CellComponentType } from './Cell.types';
 import InternalCellWithoutRT from './InternalCellWithoutRT';
 import InternalCellWithRT from './InternalCellWithRT';
 import { CellProps } from '.';
 
-const Cell = <T extends LGRowData>({
-  children,
-  cell,
-  ...rest
-}: CellProps<T>) => {
+const CellWithForwardRef = <T extends LGRowData>(
+  { children, cell, ...rest }: CellProps<T>,
+  ref: ForwardedRef<HTMLTableCellElement>,
+) => {
   return (
     <>
       {!cell && (
-        <InternalCellWithoutRT {...rest}>{children}</InternalCellWithoutRT>
+        <InternalCellWithoutRT ref={ref} {...rest}>
+          {children}
+        </InternalCellWithoutRT>
       )}
 
       {cell && (
-        <InternalCellWithRT {...rest} cell={cell}>
+        <InternalCellWithRT ref={ref} {...rest} cell={cell}>
           {children}
         </InternalCellWithRT>
       )}
     </>
   );
 };
 
+export const Cell = React.forwardRef(CellWithForwardRef) as CellComponentType;
```

Comment:
Ditto on maybe there being some value having a comment here explaining this a bit in addition to the type comment.